front/src/views/news/news.view.css

```diff
@@ -0,0 +1,76 @@
+#wrap {
+    padding-top: 20px;
+    width: 1000px;
+    margin: 0 auto;
+    font-family: var(--txt-font);
+}
+
+.row {
+    display: flex;
+    flex-direction: row;
+    box-shadow: 0px 2px 7px rgba(0, 0, 0, 0.15);   
+}
+
+.title h1 {
+    color: #000;
```

Comment:
variable? please check all your using of colors

Reply:
done